lib/utensils/database_cleaner.rb

```diff
@@ -1,6 +1,14 @@
 RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner[:mongoid].strategy = :truncation
+    DatabaseCleaner[:active_record].strategy = :transaction
```

Comment:
Do we need this (and the line below) here if it also gets set in the `around` block?